src/components/FileUpload.js

```diff
@@ -0,0 +1,40 @@
+const Data = (props) => {
+    const data = props.data;
+    const getTotalPassed = () => {
+      return data.filter((item) => item.status === "PASSED").length;
+    }
+  
+    const getTotalFailed = () => {
+      return data.filter((item) => item.status === "FAILED").length;
+    }
+  
+    const getRemaining = () => {
+      return data.filter((item) => !item.status).length;
+    }
+    return(
+      <div className='csv-data'>
+        <p>{props.data.length} pages total</p>
+        <p>{getTotalPassed()} passed, {getTotalFailed()} failed</p>
+        <p>{getRemaining()} pages left to verify</p>
+      </div>
+     )
+  }
+
+function FileUpload(props) {
+    return (
+        <div className='file-upload'>
+            <p className='instructions-message'>Prepare your CSV file with the following columns: </p>
+            <p><code>oldUrl</code>, <code>newUrl</code>, <code>status</code>, and <code>comment</code></p>
+            <div className='input-file'>
+                <input id="actual-button" type="file" name="file" onChange={props.onHandleChange} accept=".csv" hidden/>
```

Comment:
**FYI (non-blocking)**: This isn't really necessary for this tool, because using the tool requires visual interaction anyway... But hiding an input can sometimes be a little tricky because it removes it from the tab order/screen-readers.  For that reason, I usually opt for techniques like this: https://www.benmarshall.me/styling-file-inputs/

But again, that's probably overkill for this tool! Maybe as a middle ground, you can add a tab index to the label?

Reply:
> But hiding an input can sometimes be a little tricky because it removes it from the tab order/screen-readers.

This is a valid point and I agree with this! We still want it to be accessible to anyone; I have updated it here: [fa2fd4c](https://github.com/Automattic/compare-snapshot-tool/pull/5/commits/fa2fd4c1dd757d489c7c1d7eb0b05db818730ce1)

Reply:
Great catch, Dan! I learned something new!